flutter_app/lib/command_constructors/return_to_launch_constructor.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:dart_mavlink/dialects/common.dart';
+
+/// Constructs a MissionItem command to return to launch using MAV_CMD_NAV_RETURN_TO_LAUNCH (20).
+///
+/// @sequence The sequence number for the MAVLink frame.
+/// @systemId The MAVLink system ID of the vehicle (normally "1").
+/// @componentId The MAVLink component ID (normally "0").
+/// @latitude The latitude of the launchpad (Defaults to where it was armed).
+/// @longitude The longitude of the launchpad (Defaults to where it was armed).
+/// @altitude The altitude of the waypoint (Defaults to 15m)
+/// @param1 Unused
+/// @param2 Unused
+/// @param3 Unused
+/// @param4 Unused
+///
+/// @return A MissionItem representing the reutrn to launch command.
+
+MissionItem returnToLaunch(int sequence, int systemID, int componentID,
```

Comment:
Yeah, I didn't read the docs when I approved this. You need to use a Command Long for this, check set_mode_constructor.dart, that should be similar.